app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,19 @@
+<footer class="row">
+  <div class="large-12 columns"><hr/>
+    <div class="row">
+
+      <div class="large-6 columns">
+      </div>
```

Comment:
What's this empty div doing here? You shouldn't have empty divs floating around! If you're actually using it for layout purposes, look into Foundation `offsets` as described here as an alternative: http://foundation.zurb.com/docs/components/grid.html